src/ray/common/task/task.h

```diff
@@ -30,6 +31,16 @@ class Task {
     ComputeDependencies();
   }
 
+  /// Construct a vector Task. All task specs should have the same object
+  /// dependencies and resource requirements as the first.
+  Task(TaskExecutionSpecification task_execution_spec,
+       std::vector<TaskSpecification> specs)
```

Comment:
This should be `const std::vector<TaskSpecification>& specs`, otherwise there will be an additional copy.

Reply:
Really, even if you std::move twice?

Reply:
Ah ok, that should be ok!